Makefile

```diff
@@ -55,12 +55,12 @@ build-for-static-stdlib-docker:
 	@docker run \
 		-v "$(PWD):$(PWD)" \
 		-w "$(PWD)" \
-		swift:5.7-focal \
+		swift:5.8-focal \
```

Comment:
the `build-for-static-stdlib` command is what is run in CI, so we gotta update the CI steps to run in 5.8.